functests/validation/validation.go

```diff
@@ -0,0 +1,232 @@
+			Expect(err).ToNot(HaveOccurred())
+
+			err = client.Client.List(context.TODO(), mcp, &goclient.ListOptions{LabelSelector: selector})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(len(mcp.Items)).To(Equal(1))
+		})
+
+		It("should have a openshift-ovn-kubernetes namespace", func() {
+			_, err := client.Client.Namespaces().Get("openshift-ovn-kubernetes",metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			_, err = client.Client.Namespaces().Get("openshift-sdn",metav1.GetOptions{})
+			Expect(err).To(HaveOccurred())
+			Expect(errors.IsNotFound(err)).To(BeTrue())
+		})
+	})
+
+	Context("Performance", func() {
+		getPerformanceProfiles := func() []perfv1alpha1.PerformanceProfile {
+			perfProfile := &perfv1alpha1.PerformanceProfileList{}
+			err := client.Client.List(context.TODO(), perfProfile)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(len(perfProfile.Items)).To(BeNumerically(">", 0))
+
+			return perfProfile.Items
+		}
+
+		It("should have the performance operator namespace", func() {
+			_, err := client.Client.Namespaces().Get(utils.PerformanceOperatorNamespace, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("should have the performance operator deployment in running state", func() {
+			deploy, err := client.Client.Deployments(utils.PerformanceOperatorNamespace).Get(utils.PerformanceOperatorDeploymentName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(deploy.Status.Replicas).To(Equal(deploy.Status.ReadyReplicas))
+
+			pods, err := client.Client.Pods(utils.PerformanceOperatorNamespace).List(metav1.ListOptions{
+				LabelSelector: fmt.Sprintf("name=%s", utils.PerformanceOperatorDeploymentName)})
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(len(pods.Items)).To(Equal(1))
+			Expect(pods.Items[0].Status.Phase).To(Equal(corev1.PodRunning))
+		})
+
+		It("Should have the performance CRD available in the cluster", func() {
+			crd := &apiext.CustomResourceDefinition{}
+			err := client.Client.Get(context.TODO(), goclient.ObjectKey{Name: utils.PerformanceCRDName}, crd)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("should have a performance policy applied in the cluster", func() {
+			getPerformanceProfiles()
+		})
+
+		// TODO: this can be removed if already validated in the performance test suite
+		It("should have a real time kernel on the node if requested by the profile", func() {
+			profiles := getPerformanceProfiles()
+			for _, profile := range profiles {
+				if profile.Spec.RealTimeKernel != nil && profile.Spec.RealTimeKernel.Enabled == pointer.BoolPtr(true) {
+					nodes, err := client.Client.Nodes().List(metav1.ListOptions{LabelSelector: labels.Set(profile.Spec.NodeSelector).String()})
+					Expect(err).ToNot(HaveOccurred())
+					for _, node := range nodes.Items {
+						Expect(strings.Contains(node.Status.NodeInfo.KernelVersion, "rt")).To(BeTrue())
+					}
+				}
+			}
+		})
+	})
+
+	Context("SRIOV", func() {
```

Comment:
Same, lowercase as in the sriov suite. In this way skipping / focusing will work as well.